full-moon/src/ast/parsers.rs

```diff
@@ -1383,6 +1385,32 @@ cfg_if::cfg_if! {
                 ))
             }
         );
+
+        // Roblox Compound Assignment
+        #[derive(Clone, Debug, Default, PartialEq)]
+        struct ParseCompoundAssignment;

```

Comment:
Alphabetical order with the rest, so that `ParseCompoundAssignment` is not after `ParseTypeSpecifier`.